Makefile

```diff
@@ -34,17 +38,26 @@ deploy-s2:
 	git checkout master
 
 deploy-gh-pages:
-	(LONG_CACHE=true BASE_URL=https://img.shields.io npm run build && \
-	git checkout -B gh-pages master && \
-	cp build/index.html index.html && \
-	cp -r build/_next next && \
-	pushd next/*/page && mv {_,}error && popd && \
-	sed -i 's,/_next/,./next/,g' index.html $$(find next -type f) && \
-	sed -i 's,_error,error,g' index.html $$(find next -type f) && \
-	git add -f build index.html next && \
-	git commit -m '[DEPLOY] Build index.html' && \
-	git push -f origin gh-pages:gh-pages) || git checkout master
-	git checkout master
+	rm -rf ${DEPLOY_TEMP}
+	git worktree prune
+	LONG_CACHE=true \
+		BASE_URL=https://img.shields.io \
+		NEXT_ASSET_PREFIX=https://shields.io \
+		npm run build
+	git worktree add -B gh-pages ${DEPLOY_TEMP}
+	git -C ${DEPLOY_TEMP} ls-files | xargs git -C ${DEPLOY_TEMP} rm
+	git -C ${DEPLOY_TEMP} commit -m '[DEPLOY] Completely clean the index'
+	cp -r build/* ${DEPLOY_TEMP}
+	cp favicon.png ${DEPLOY_TEMP}
+	echo shields.io > ${DEPLOY_TEMP}/CNAME
```

Comment:
The point of the CNAME file was to have GitHub redirect from badges.github.io/shields to shields.io.

I believe it no longer needs that, as it is now set up through the project settings: https://help.github.com/articles/adding-or-removing-a-custom-domain-for-your-github-pages-site/

Reply:
Something in my testing suggested the file still has an effect. When I ran this script on my fork without the CNAME file in gh-pages, it turned off the custom domain.

This link suggests changes made in the UI are stored in the repo: https://help.github.com/articles/troubleshooting-custom-domains/#github-repository-setup-errors